lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -457,6 +474,38 @@ withSizeHeader16be x = do
     putWord16be (fromIntegral $ BS.length bs)
     putByteString bs
 
+-- | Convert an 'Integral i' to 'Int'. Doesn't /expect/ overflow, but uses
+-- 'overflowSafeCoerce' internally.
+toInt :: (Show i, Typeable i, Integral i) => i -> Int
+toInt = overflowSafeCoerce
```

Comment:
Hmmm... this defy a bit the purpose of having the overflowSafeCoerce function. Note that there's a point in our style guide about type-aliases and, it doesn't help readability and here, it hides crucial details. The calling code should be calling some `unsafe` coercion function, and keep the `unsafe` word well visible to convey the right meaning.